src/main/java/com/eerussianguy/blazemap/feature/atlas/AtlasTask.java

```diff
@@ -76,7 +78,8 @@ public enum Stage {
         QUEUED,
         CALCULATING,
         STITCHING,
-        SAVING
+        SAVING,
+        COMPLETE
```

Comment:
I'm not unhappy about the COMPLETE stage... but the reason there wasn't one is because the user will never see it, at most it'll be visible during exactly one frame that lasts as much as 16ms (faster than you can notice it changed) if you play at only 60fps. The reason stages exist is for debugging purposes on the slower stages. The only exception is QUEUED that will only show if something ever goes very horribly wrong with the background threads.

This is just a note, not a thing that needs to be changed. You added it, it's added. It is a thing now.

Reply:
I was contemplating not adding it and just tacking it onto the end of the saving section. That is still an option if it's a better approach.